MCprep_addon/README.md

```diff
@@ -0,0 +1,362 @@
+### Utilities 
```

Comment:
Either the filename or location seems off. README generally refers to a user, whereas `CONTRIBUTING.md` in the root level is for developers. Since this is developer documentation, I think it should probably [live here](https://github.com/TheDuckCow/MCprep/tree/master/docs) and call it dev utils, but open to other ideas.

Reply:
Okay I rename and move it there for now.